introduction.md

```diff
@@ -10,13 +10,15 @@ position: 0
 
 # Welcome to Telerik Document Processing Libraries
 
-Telerik Document Processing is a bundle of UI-independent, cross-platform libraries enabling you to process the most commonly used flow, fixed and spreadsheet document formats. The components allow to create, import, modify and export documents, without external dependencies. To read more about the benefits of using Telerik Document Processing, please visit the [product overview page](https://www.telerik.com/document-processing-libraries).
+Telerik Document Processing is a bundle of UI-independent cross-platform libraries that enable you to process the most commonly used text, PDF, and spreadsheet file formats. The Document Processing library allows you to create, import, modify and export documents without relying on external dependencies like Adobe Acrobat or Microsoft Office.
```

Comment:
- To me, _UI-independent_ sounds like a subordinate to cross-platform, so there must be no comma here.
 - Let's get straight to the point and call the supported file types with their real names instead of hiding them behind terms and collective nouns.
 - Please don't use _please_ in docs.